packages/spotify_ml/spotify_ml.push/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Spotify API bindings"
+description: "Interact with the Spotify Web API via OCaml."
+maintainer: ["maxrn"]
+authors: ["maxrn"]
+license: "MIT"
+tags: ["spotify"]
+homepage: "https://github.com/maxrn/spotify_ml"
+bug-reports: "https://github.com/maxrn/spotify_ml/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.16"}
+  "ocamlformat" {dev}
+  "serde"
+  "serde_derive"
```

Comment:
Why do I need a version constraint on exactly this package and not to e.g. `serde`?

Reply:
The error that the CI found was during the ppx/deriving part of the compilation. Hence, I added the constraint to the deriving package of the serde "suite".

That being said, `serde_deriving` depends on `serde` with [the version constrain `{=version}`](https://github.com/ocaml/opam-repository/blob/master/packages/serde_derive/serde_derive.0.0.2/opam#L14). So in practice, adding a constraint for the deriving package also adds a constraint on the main package, by transitivity.

(It would probably make sense to also add the constraint on the dependency to `serde`. But I lack the domain-specific knowledge to know for sure. So I did the smallest fix.)

Reply:
I guess I'm confused, because looking at the build log it seems like during the first install opam installed the correct versions of everything (0.0.2 in serde's case), but then reinstalled my package and that is when it downgraded the versions? I guess for the future I will just ping the exact versions my package depends on to avoid those errors.

Anyway, thanks for fixing it and merging my PR!